integration-tests/sql/README.adoc

```diff
@@ -42,4 +42,32 @@ $Env:SQL_JDBC_USERNAME="#username"
 $Env:SQL_JDBC_PASSWORD="#password"
 ```
 
-Oracle database could be used as external db. In that case use parameter `-Dcq.sqlJdbcKind=oracle`.
\ No newline at end of file
+Oracle database could be used as external db. In that case use parameter `-Dcq.sqlJdbcKind=oracle`.
+
+=== External Derby database
+
+To execute tests against external Derby database, stored procedure has to be uploaded into the database classpath.
+Jar with stored procedure for the derby database is creaed by module `sql-derby`.
+Jar could be uploaded via following commands through `ij`:
+```
+CALL sqlj.install_jar('/PATH_TO_JAR/camel-quarkus-integration-test-sql-derby-stored-procedure-*.jar', 'AddNumsProcedure' , 0)
+
+CALL syscs_util.syscs_set_database_property('derby.database.classpath', 'APP.ADDNUMSPROCEDURE')
+```
```

Comment:
In such case, I wonder whether pointing to a documentation owned by derby would be good ? From time to time, I found us modifying docs for things that under camel-quarkus control.

Reply:
I agree that linkto some doc would help us to avoid modyfication od this readme. But on the other hand, the same code is used in docker initialization in this change. So we would need to keep it updated either way. (But of course in case that there is no better way of initialization of stored procedure in derby, it is still worth of investigation)

Reply:
Indeed, it's clear now that this is the itest doc and not the extension doc. So, yes I also think we need to keep this.